deploy/helm-chart/templates/_helpers.tpl

```diff
@@ -33,4 +33,22 @@ Allow the release namespace to be overridden
   {{- else -}}
     {{- .Release.Namespace -}}
   {{- end -}}
-{{- end -}}
\ No newline at end of file
+{{- end -}}
+
+{{/*
+Generate labels to be used
+*/}}
+{{- define "promscale.labels" -}}
+app: {{ include "promscale.fullname" . }}
+chart: {{ template "promscale.chart" . }}
+release: {{ .Release.Name }}
+heritage: {{ .Release.Service }}
+{{- end }}
+
+{{- define "promscale-helm.labels" -}}
+{{ include "promscale.labels" . }}
+app.kubernetes.io/name: {{ include "promscale.fullname" . | quote }}
+app.kubernetes.io/version: {{ .Chart.AppVersion }}
+app.kubernetes.io/part-of: {{ include "promscale.fullname" . | quote }}
```

Comment:
We should remove this label as this helm chart is shipping a stack of applications.

As per https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels